tests/Integration/Worse/WorseIndexMemoryBuilderTest.php

```diff
@@ -19,4 +19,9 @@ protected function createBuilder(Index $index): IndexBuilder
         );
         return $indexBuilder;
     }
+
+    public function testIndexesClassLike(): void
+    {
+        $this->markTestSkipped();
```

Comment:
This is the legacy indexer, but this was only supposed to prevent the trait test running -- not all of them.